stripe/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -752,5 +753,18 @@ data class PaymentMethodCreateParams internal constructor(
                 )
             )
         }
+
+        @JvmStatic
+        @JvmOverloads
+        fun createBlik(
+            billingDetails: PaymentMethod.BillingDetails? = null,
```

Comment:
Do billing details apply for Blik?

Reply:
it's not explicitly requested by blik team, but I found [(empty)billing details](https://git.corp.stripe.com/gist/ccen/cd018016c6601a7ed3de9c504734348c) are returned when creating a blik payment method - does that mean we should allow billing details as create param? can remove otherwise

Reply:
You can sync with @yuki-stripe but likely should be excluded.

Reply:
just checked with yuki, sounds like this should be included:)

Reply:
@mshafrir A user should be able to create **any** PaymentMethod with billing details, what am I missing here?

Reply:
@yuki-stripe looks like Paypal and Alipay don't support billing details on the create path. Is this inconsistent with iOS?

Reply:
@mshafrir Yes it is. That's odd